src/components/learner-credit-management/MemberStatusTableCell.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Badge } from '@edx/paragon';
+
+import {
+  Timelapse, CheckCircle, RemoveCircle,
+} from '@edx/paragon/icons';
+
+const MemberStatusTableCell = ({
+  row,
+}) => {
+  let icon;
+  if (row.original.status === 'pending') {
```

Comment:
This formatting of the chips is not consistent with the other example in Learner Credit, we should be mimicking the same look of the ones [here](https://github.com/openedx/frontend-app-admin-portal/blob/8e145f93780545ed352fe6f091e677b84c46fe6d/src/components/learner-credit-management/assignments-status-chips/NotifyingLearner.jsx#L9)